src/content/components/FlameChartCanvas.js

```diff
@@ -267,7 +267,8 @@ class FlameChartCanvas extends PureComponent {
       const start = stackTiming.start[i];
       const end = stackTiming.end[i];
       if (start < time && end > time) {
-        return { depth, stackTableIndex: i };
+        const timing: HoveredStackTiming = { depth, stackTableIndex: i };
+        return timing;
```

Comment:
nit: this change is not really useful given that the function return type is already typed